egs/librispeech/s5/run.sh

```diff
@@ -355,6 +355,10 @@ local/run_cleanup_segmentation.sh
 # train NN models on the entire dataset
 local/nnet2/run_7a_960.sh
```

Comment:
@danpovey We could probably remove nnet2 scripts from run.sh as we no longer recommend them; what do you think ?